GWAS/PCA.ipynb

```diff
@@ -595,19 +695,19 @@
    },
    "outputs": [],
    "source": [
-    "# Calculate Mahalanobis distance per population and report outliers\n",
+    "##### Calculate Mahalanobis distance per population and report outliers\n",
```

Comment:
Why `#####`? Please use one `#`. Look at the `sos run PCA.ipynb -h` output. This matters to the contents displayed there.